VirtoCommerce.PricingModule.Web/Scripts/blades/assignment-list.js

```diff
@@ -84,6 +82,29 @@ function ($scope, assignments, dialogService, uiGridHelper, bladeUtils, catalogs
         dialogService.showConfirmationDialog(dialog);
     }
 
+    $scope.deleteAllFiltered = function () {
+        searchAssignments(0, Number.MAX_VALUE, function (data) {
+            var filteredIds = _.pluck(data.results, 'id');
+            var dialog = {
+                id: "confirmDeleteItems",
+                itemCount: filteredIds.length,
+                callback: function (confirm) {
+                    if (confirm) {
+                        closeChildrenBlades();
+                        blade.isLoading = true;
+
+                        assignments.remove({ ids: filteredIds }, function () {
+                            blade.refresh();
+                        }, function (error) {
```

Comment:
Explicit error handling is not needed, as we have global interceptor showing error details.

Reply:
Done